deploy/Provisioning-1674548289785.json

```diff
@@ -746,7 +747,7 @@
               {
                 "id": "color",
                 "value": {
-                  "fixedColor": "red",
+                  "fixedColor": "orange",
```

Comment:
I don’t like red color for p50 dots, red means "bad" so I am proposing orange here. No other colors from the default color palettes are available.